pages/coaching-dashboard.tsx

```diff
@@ -48,13 +90,14 @@ const coachingDashboard = () => {
   return (
     <div className="flex flex-col p-4 m-4 ">
       <p className="mb-8 text-3xl font-bold">Coaching Dashboard</p>
-      <h2 className="mb-4">Enrolled Students</h2>
       <div className="grid grid-cols-1 gap-4 sm:grid-cols-3">
         {userList.map((it, index) => {
+          const completionDate = goalCompletionDateList[index];
+          const completedGoal = completedGoalsList[index];
           return (
             <div key={index}>
```

Comment:
you can actually get rid of this div and put the same key prop inside the Link component right below